cmd/run.go

```diff
@@ -294,8 +295,14 @@ func serviceRun(ctx *cli.Context) error {
 	for _, env := range ctx.StringSlice("env") {
 		parts := strings.SplitN(env, "=", 2)
 		value := ""
+
 		if len(parts) > 1 {
 			value = parts[1]
+
+			if len(parts[0]) == 0 {
+				errMsg := fmt.Sprintf("invalid argument \"%s\" for e: invalid environment variable: %s\nSee 'rancher run --help'.", env, env)
```

Comment:
I follow docker's error message on that, if we run: docker run -e "=bar" ubuntu, the msg will be:
invalid argument "=bar" for e: invalid environment variable: =bar
See 'docker run --help'.